docs-src/multiple-ns.md

```diff
@@ -0,0 +1,32 @@
+# Routes in multiple namespaces
+
+With the Service APIs, a single Gateway can target routes across multiple
+namespaces. 
+
+This guide assumes that you have installed Service APIs CRDs and a conformant
+controller.
+
+In the following example:
+
+- `acme-lb` GatewayClass: The GatewayClass responsible for satisfying Gateway
+  and Route resources.
+- `multi-ns-gateaway` Gateway: The Gateway is configured with a single listener
+  on port 80 which selects routes from **All namespaces** which have the label
+  `product: baz`. Notice how the `routes.namespaces.from` field in the listener
+  is set to `All`.
+- `foo-ns` and `bar-ns` Namespaces: These are two namespaces in which we route
+  resources are installed.
+- `http-app-1` and `http-app-2` HTTPRoutes: These are two resources that are
+  installed in separate namespaces. These routes will be picked up by
+  `multi-ns-gateway` resource for two reasons:
+    - Both have the `product: baz` label on them.
+    - Both have `spec.gateways.allow` set to `All`. The route owner needs to
+      explictly set this field so that Gateways from other namespaces can target
+      the route.
```

Comment:
Can we use this example to show both approaches? One with `allow: All` and one with `allow: FromList`. Unfortunately that would also require creating a namespace for the Gateway.

Reply:
> Unfortunately that would also require creating a namespace for the Gateway.

I'll put the Gateway in the default namespace.